app/managers/session_manager.ts

```diff
@@ -190,10 +192,11 @@ class SessionManager {
 
             relaunchApp({launchType, serverUrl, displayName});
         }
+        this.terminatingSessionUrl.delete(serverUrl);
```

Comment:
Maybe a setTimeout here could also help?

Reply:
Not sure how that will help. I don't expect anything inside this function to crash or take an infinite amount of time, so... a timeout would only cause problems if things get executed out of place. But 0/5.